app/routes/press-releases/press-release.js

```diff
@@ -2,16 +2,9 @@ import Route from '@ember/routing/route';
 import PressReleaseSnapshot from '../../utils/press-release-snapshot';
 
 export default class PressReleasesPressReleaseRoute extends Route {
-  beforeModel(transition) {
-    if(transition.from) {
-      this.fromRoute = transition.from.name;
-    } else {
-      this.fromRoute = 'press-releases.overview.concept';
-    }
-  }
-
   async model(params) {
-    const pressRelease = await this.store.findRecord('press-release', params.press_release_id, {
+    // const pressRelease =
+    return await this.store.findRecord('press-release', params.press_release_id, {
```

Comment:
Je hebt 2 return statements in deze `model`-hook. 

De `model`-hook mag ook een promise returnen. Daar wordt dan automatisch op gewacht door Ember. De `await` is hier dus niet noodzakelijk bij het `return` statement (en de `async` kan dan ook weg).